### PR TITLE
Drop menu Homework

### PR DESCRIPTION
**Pull request by dragan-andrei**
# My DropMenu Homework

### This is how my menu looks like.  This work was aneasy, because you need to use a lot of parental connections and e.t.c. It is very easy to get confused xDD)) I've done shadow to *"Logo"*, because it's look very good and attractive!
image

### CODE REVIEW

**Comment by IvanCeban** [comment]
Looks great! You're right, it's easy to get lost in parental connections. That was my point, and that's what should become natural to you when you gain more experience ;)

File: .gitignore [hunk 1]
```diff
@@ -1,3 +1,6 @@
+<<<<<<< HEAD
```
Comment (IvanCeban) [inline .gitignore]: you've got conflicts here, take care to remove the generated redundant text

**Review by IvanCeban — APPROVED** [review]
Great job!

**Review by CozlovschiNichita — APPROVED** [review]
при наведении нет  **underline** :D

**Comment by dragan-andrei** [comment]
> при наведении нет  **underline** :D

image

Посмотри в Задание. Подчеркивание попросили только на Home, About. Products & Services просто меняют цвет

**Comment by IvanCeban** [comment]
> при наведении нет **underline** :D

howkeye

**Pull request merged**